app/assets/javascripts/services/event_notifications_service.js

```diff
@@ -299,9 +299,12 @@ angular.module('ManageIQ').service('eventNotifications', ['$timeout', '$window',
     }
   };
 
-  this.showToast = function(notification, persist) {
+  this.showToast = _.throttle(function(notification, persist) {
     var $this = this;
     notification.show = true;
+    if (state.toastNotifications.length > 2) {
+      state.toastNotifications.splice(0, 1);
```

Comment:
remove the first element of the array if it's long?

* isn't this just `.unshift()`? (or `.pop()`?)
* did you mean something like `.length = 2` instead?

Reply:
Yeah :laughing: you know my :heart: towards JS :laughing: gonna fix